projects/1.16/assets/just-enough-effect-descriptions-jeed/jeed/lang/zh_cn.json

```
@@ -1,8 +1,43 @@
 {
-  "jeed.category.effect_info": "效果",
-  "jeed.tooltip.color_complete": "%1$s：%2$s",
-  "jeed.tooltip.color": "颜色",
-  "jeed.tooltip.beneficial": "有益",
-  "jeed.tooltip.harmful": "有害",
-  "jeed.description.missing": "暂无描述"
+    "jeed.category.effect_info": "效果",
+    "jeed.tooltip.color_complete": "%1$s：%2$s",
+    "jeed.tooltip.color": "颜色代码",
+    "jeed.tooltip.beneficial": "增益效果",
+    "jeed.tooltip.harmful": "负面效果",
+    "jeed.description.missing": "无可用描述",
+  
+    "effect.minecraft.absorption.description": "给予吸收伤害用生命值（不会自然恢复）；等级越高，给予的生命值越高。",
+
+    "effect.minecraft.bad_omen.description": "进入村庄后引发袭击；等级越高，袭击的难度越大。",
+    "effect.minecraft.blindness.description": "视力受限；且无法疾跑和暴击。",
+    "effect.minecraft.darkness.description": "导致视力暂时恶化。",
+    "effect.minecraft.conduit_power.description": "水下视力及水下挖掘速度增加，且不会溺水。",
+    "effect.minecraft.dolphins_grace.description": "海豚大幅提高玩家的游泳速度。",
+    "effect.minecraft.fire_resistance.description": "获得对火焰、熔岩伤害的免疫。",
+    "effect.minecraft.glowing.description": "使实体发光，并带有可以穿过非透明方块看到的轮廓线。",
+    "effect.minecraft.haste.description": "提高挖掘和攻击速度；等级越高，速度越快。",
+    "effect.minecraft.health_boost.description": "获得额外生命值上限；等级越高，上限越大。",

```

```suggestion
    "effect.minecraft.health_boost.description": "提升生命上限；等级越高，上限越大。",

```
来源：[生命提升](https://minecraft.fandom.com/zh/wiki/%E7%94%9F%E5%91%BD%E6%8F%90%E5%8D%87)